runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -287,14 +305,28 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 	@Override
 	public void makeWorldOverlayHint(Graphics2D graphics, ClueScrollPlugin plugin)
 	{
-		LocalPoint localLocation = LocalPoint.fromWorld(plugin.getClient(), getLocation());
+		LocalPoint localPoint = LocalPoint.fromWorld(plugin.getClient(), getLocation());
 
-		if (localLocation == null)
+		if (localPoint != null)
 		{
-			return;
+			OverlayUtil.renderTileOverlay(plugin.getClient(), graphics, localPoint, plugin.getEmoteImage(), Color.ORANGE);
 		}
 
-		OverlayUtil.renderTileOverlay(plugin.getClient(), graphics, localLocation, plugin.getEmoteImage(), Color.ORANGE);
+		final WorldPoint[] worldPoints = stashUnit.getWorldPoints();
+
+		for (final WorldPoint worldPoint : worldPoints)
+		{
+			final LocalPoint stashUnitLocalPoint = LocalPoint.fromWorld(plugin.getClient(), worldPoint);
+
+			if (stashUnitLocalPoint != null)
+			{
+				Polygon poly = Perspective.getCanvasTilePoly(plugin.getClient(), stashUnitLocalPoint);
+				if (poly != null)
+				{
+					OverlayUtil.renderPolygon(graphics, poly, Color.RED);
```

Comment:
IMO this should highlight the stash unit object and not tile

Reply:
Agreed, and i will change it later, for now adam said to leave it as is:
https://discordapp.com/channels/301497432909414422/419891709883973642/580862420659208223